docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -207,7 +207,7 @@ Returns all information associated with the account of provided Pubkey
 fields:
   - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
   - `encoding: <string>` - encoding for Account data, either "base58" (*slow*), "base64", "base64+zstd", or "jsonParsed".
-    "base58" is limited to Account data of less than 128 bytes.
+    "base58" is limited to Account data of less than 129 bytes.
```

Comment:
Can you edit this comment too? https://github.com/solana-labs/solana/blob/9a7cd8885d475dca03efce30c055f0c61e1db866/core/src/rpc.rs#L1577
Also the `encoding` docs for `getMultipleAccounts` and `getProgramAccounts` in this file

Reply:
Done